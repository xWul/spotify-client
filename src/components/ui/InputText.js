import React from 'react'
import styled from 'styled-components'

const Content = styled.div`
  width: 100%;
  padding: 5px;
`

const Input = styled.input`
  background-color: #FFF;
  padding: 6px;
  width: 100%;
  border: 1px solid #333;
  border-radius: 3px;
  font-size: 18px;
`

const InputText = (props) => {
  return (
    <Content>
      <Input type={'text'} {...props}/>
    </Content>
  )
}

export default InputText
