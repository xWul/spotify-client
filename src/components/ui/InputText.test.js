import React from 'react'
import renderer from 'react-test-renderer'
import InputText from './InputText'

test('InputText renderer without crash', () => {
  const component = renderer.create(
    <InputText placeholder={'testing'}/>,
  )

  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
