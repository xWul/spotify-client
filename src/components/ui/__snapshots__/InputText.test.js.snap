// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputText renderer without crash 1`] = `
<div
  className="sc-bdVaJa LDKxg"
>
  <input
    className="sc-bwzfXH cTikch"
    placeholder="testing"
    type="text"
  />
</div>
`;
